src/main/frontend/components/NavBar.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from "react";
+import { Switch, Route, Link, Redirect } from "react-router-dom"
+
+import ConcertVenueForm from "./ConcertVenueForm"
+import VenueIndex from "./VenueIndex.js"
+
+const NavBar = props => {
+
+
+
+return(
+<div>
```

Comment:
Again, please cleanup indentation and whitespace -- you should have a maximum of one empty line between lines 7 and 11